dashing/distribution.yaml

```diff
@@ -160,6 +160,17 @@ repositories:
       url: https://github.com/AprilRobotics/apriltag.git
       version: master
     status: maintained
+  behaviortree_cpp_v3:
+    release:
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/BehaviorTree/behaviortree_cpp_v3-release.git
+      version: 3.0.8-1
+    source:
+      type: git
+      url: https://github.com/BehaviorTree/BehaviorTree.CPP.git
+      version: master
```

Comment:
@facontidavide This branch name seems to be wrong. Should this be `ros2-dev` instead?

There seem to be numerous differences between the `master` and `ros2-dev` branch can you please clarify which branch you indent to use for ROS 2 development / releases?

Reply:
ros/rosdistro#21342 switches the devel job to the suggested branch `ros2-dev`.

Reply:
thanks. I messed up!